test/12_nullablearray.jl

```diff
@@ -965,4 +965,66 @@ for isordered in (false, true)
     end
 end
 
+
+# Tests of vcat of NullableCategoricalArray
+
+for (CA,A) in ((:NullableCategoricalArray,:NullableArray),)
```

Comment:
OK. Then if you want just move these to a `13_arraycommon.jl` file (or something similar). We'll progressively move more common tests there.